routes/applications.js

```diff
@@ -14,16 +14,12 @@ router.get("/applications", async (req, res) => {
     }
 });
 
-
-
 router.post("/applications", async (req, res) => {
-    const { userId, text } = req.body;
-    const date = new Date();
-
+    const { name, text } = req.body;
```

Comment:
I guess this will change after we confirm the ERD

Reply:
absolutely